doc/using_azure-iot-sdk-c_with_the_latest_tls_version_only.md

```diff
@@ -0,0 +1,52 @@
+# How to configure the Azure IoT C SDK and currently supported TLS platforms to use TLS 1.2 only 
+
+
+## SChannel (Microsoft Windows)
+
+To use exclusively TLS 1.2 in Microsoft Windows using SChannel, disable the support for TLS 1.0 and 1.1.
+
+Be aware that this action might impact other applications in the systems that might not support higher versions of TLS.
+
+To disable support in SChannel for TLS 1.0, follow this [documentation](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/operations/manage-ssl-protocols-in-ad-fs#enable-and-disable-tls-10).
+
+To disable support in SChannel for TLS 1.1, follow this [documentation](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/operations/manage-ssl-protocols-in-ad-fs#enable-and-disable-tls-11).
+
+
+## OpenSSL
+
+The latest Azure IoT C SDK uses [TLS 1.2 by default with OpenSSL](https://github.com/Azure/azure-c-shared-utility/blob/48f7a556865731f0e96c47eb5e9537361f24647c/adapters/tlsio_openssl.c#L1167).
+
+The SDK creates the SSL context using the method related to the TLS version selected (1.2 by default).
```

Comment:
Note: This _forces_ TLS1.2 and will not allow for automatic TLS1.3 upgrade in the future. OpenSSL 1.1.1 added support for TLS1.3 (https://wiki.openssl.org/index.php/TLS1.3)

Reply:
We should add that as a note, since would require code change.